awx/main/scheduler/task_manager.py

```diff
@@ -587,4 +587,4 @@ def schedule(self):
 
                 # Operations whose queries rely on modifications made during the atomic scheduling session
                 for wfj in WorkflowJob.objects.filter(id__in=finished_wfjs):
-                    wfj.send_notification_templates('succeeded' if wfj.status == 'successful' else 'failed')
+                    wfj.send_notification_templates('started' if wfj.status == 'running' else ('succeeded' if wfj.status == 'successful' else 'failed'))
```

Comment:
I don't think this is where this belongs.  I think this code is explicitly handling workflow jobs that *finished*?  So this is probably _not_ where we want to send "on start" notifications cc @chrismeyersfsu

Reply:
If we add the on-start notifications to that line, then we should move this _existing code_ for the failed / success case up to around:

https://github.com/ansible/awx/blob/b1d75327e3b6bb146da8761f285f6b45608505ad/awx/main/scheduler/task_manager.py#L195

I've sat next to them while @chrismeyersfsu and @beeankha were troubleshooting this. I won't spoil the ending for you, but I'm pretty sure the trouble they are having is the same reason that _this_ code was put in this place instead of the more logical location.